docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    precision_mats = cmn.predict_precisions(C)
</pre><p>Silence these outputs with %%capture at the start of fitting cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>